jbpm-persistence/jbpm-persistence-api/src/main/java/org/jbpm/persistence/api/integration/model/TaskInstanceView.java

```diff
@@ -70,8 +74,19 @@
     public TaskInstanceView() {        
     }
     
-    public TaskInstanceView(Task source) {
```

Comment:
Doesn't it break backward compatibility?

Reply:
not really, TaskInstanceView is internal and this constructor was only used where it was modified